File: 1768-merge-strings-alternately/1768-merge-strings-alternately.cpp
class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        std::string merge = "";
        int l = std::min(word1.length(), word2.length());
        for(int i = 0; i < l; ++i){
            merge = merge + word1[i] + word2[i];
        }
        return merge + word1.substr(l) + word2.substr(l);
    }
};

File: 1768-merge-strings-alternately/1768-merge-strings-alternately.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        merge = ""
        l = min(len(word1), len(word2))
        for i in range(l):
            merge += word1[i] + word2[i]
        return merge + word1[l:] + word2[l:]